packages/eds-core-react/src/components/Popover/Popover.docs.mdx

```diff
@@ -107,6 +107,14 @@ const Demo = () => {
 
 <Story id="data-display-popover--with-close-button"/>
 
+### Not dismissable
+
+This story shows `Popover` with more complex content. <br />
+By ignoring `onClose` on Popover, and button setting `open` state to `false`, 
+this will give more control over closing Popover.
```

Comment:
Maybe something like: "To prevent dismissing popover with click outside or escape key, simply leave out the onClose callback"